app/javascript/sdk/IFrameHelper.js

```diff
@@ -36,13 +36,22 @@ import {
 } from 'shared/helpers/AudioNotificationHelper';
 import { isFlatWidgetStyle } from './settingsHelper';
 import { popoutChatWindow } from '../widget/helpers/popoutHelper';
+import { LocalStorage } from 'shared/helpers/localStorage';
 
 const updateAuthCookie = cookieContent =>
   Cookies.set('cw_conversation', cookieContent, {
     expires: 365,
     sameSite: 'Lax',
   });
 
+const updateCampaignReadStatus = campaignId => {
+  const campaignsRead = LocalStorage.get('campaigns_read_by_id') || {};
+  LocalStorage.set('campaigns_read_by_id', {
```

Comment:
@nithindavid I'm not sure about the implementation here. A couple of questions.

1/ Why is the local storage instead of a cookie?
2/ I don't see the expiry of being checked anywhere.
3/ This implementation would not fix the issue. If I'm on page 1, I close campaign 1. If I'm on page 2, campaign 2 would open. 

The reported issue was that if the user has dismissed the campaigns, we should not show the campaigns again for a reasonable time.

Reply:
Thanks for the feedback. I'll check and revert soon 👍

Reply:
@pranavrajs I have made the changes. Currently it'll snooze all campaigns for an hour. Let me know if it should be different.